### PR TITLE
Added icon for IP Addresses in Available devices

### LINKED ISSUE #19: Add Icon before IP Address shown on Discover page
**Note:** This issue is reserved for SLoP Contributors.

The icon used here is from [Feather Icons](https://feathericons.com/). You can either use dart package for feather icon or .svg itself.

Android - 2

### PR DESCRIPTION
**Pull request by ritvij14**
## Related issues
Fixes #19 

## Description
- Added a `cast` icon before IP Addresses. I have added the feather icon package in the project for future use as well.
- Also removed an extra instance of the IP Address list which caused the second list of the same IP Addresses.

## Output
WhatsApp Image 2020-10-05 at 7 03 08 PM

### CODE REVIEW

File: lib/pages/discover.dart [hunk 1]
```diff
@@ -103,12 +96,17 @@ class Device extends StatelessWidget {
...
     return GestureDetector(
-      child: Text(
-        ipAddr,
-        style: TextStyle(
-          color: Sinix.textColor,
-          fontSize: 18,
-        ),
+      child: Row(
+        children: <Widget>[
+          Icon(FeatherIcons.cast),
```
Comment (sanket143) [inline lib/pages/discover.dart:100]: Add some space between the icon and the IP address

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: lib/pages/discover.dart (hunk 1 above)
Comment (ritvij14) [reply lib/pages/discover.dart:100]: @sanket143 done, I have added a `SizedBox`

File: lib/pages/discover.dart [hunk 2]
```diff
@@ -24,7 +24,7 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
 
     final stream = NetworkAnalyzer.discover2(subnet, port);
 
-    deviceList = [];
+    deviceList = ['192.168.1.1'];
```
Comment (sanket143) [inline lib/pages/discover.dart]: You forgot to remove the test element.

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: lib/pages/discover.dart (hunk 2 above)
Comment (ritvij14) [reply lib/pages/discover.dart]: @sanket143 yeah sorry about that one :sweat_smile: removed it

**Review by sanket143 — APPROVED** [review]

**Comment by sanket143** [comment]
give @ritvij14 25 points

**Pull request merged**